### DISCUSSION TITLE
Add Linter rules to reduce code review time

### DISCUSSION THREAD

**Discussion by doannimble** [nimblehq/flutter_templates#269] [RFCs]
## Issue

We usually make the mistake of lint check such as missing trailing commas, splash for document comments, preferring single quotes, etc...
It will increase the code review time since we need more effort to review and fix these.

## Solution

Following https://dart.dev/tools/linter-rules, we can pick and enable appropriate lint check rules. Then run `dart fix --apply` to do the quick fix.

## Who Benefits?

Developers

## What's Next?

- [ ] Enable lint check rules at `analysis_options.yaml` by https://dart.dev/tools/linter-rules
- [ ] Update README to guide how to do the quick fix by running `dart fix --apply` (https://dart.dev/tools/dart-fix)

**Comment by luongvo** [comment]
I voted for `All` to enable it at least. However, we can re-check this rule list in the final implementation.
